solutions/swb-reference/SWBv2.postman_collection.json

```diff
@@ -1118,37 +1118,107 @@
       ]
     },
     {
-      "name": "Create Hosting Account",
-      "request": {
-        "method": "POST",
-        "header": [
-          {
-            "key": "Cookie",
-            "value": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}",
-            "type": "text"
+      "name": "hosting accounts",
```

Comment:
Didn't want to dig for it, but do we also have Update and Get Hosting Account?

Reply:
Nope, we missed those in those PRs 🤡 . Added.